instrumentation/pg/lib/opentelemetry/instrumentation/pg/constants.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module PG
+      module Constants
+        # A list of SQL commands, from: https://www.postgresql.org/docs/current/sql-commands.html
+        # Commands are truncated to their first word, and all duplicates
+        # are removed, This favors brevity and low-cardinality over descriptiveness.
+        SQL_COMMANDS = %w[
+          ABORT
+          ALTER
+          ANALYZE
+          BEGIN
+          CALL
+          CHECKPOINT
+          CLOSE
+          CLUSTER
+          COMMENT
+          COMMIT
+          COPY
+          CREATE
+          DEALLOCATE
+          DECLARE
+          DELETE
+          DISCARD
+          DO
+          DROP
+          END
+          EXECUTE
+          EXPLAIN
+          FETCH
+          GRANT
+          IMPORT
+          INSERT
+          LISTEN
+          LOAD
+          LOCK
+          MOVE
+          NOTIFY
+          PREPARE
+          PREPARE
+          REASSIGN
+          REFRESH
+          REINDEX
+          RELEASE
+          RESET
+          REVOKE
+          ROLLBACK
+          SAVEPOINT
+          SECURITY
+          SELECT
+          SELECT
+          SET
+          SHOW
+          START
+          TRUNCATE
+          UNLISTEN
+          UPDATE
+          VACUUM
+          VALUES
+        ].freeze
```

Comment:
I don't honestly know if all of these are valuable, but I'm erring on the side of including them until we have a better answer for tuning instrumentation noise.